.github/workflows/test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
   smir-integration-tests:
     needs: code-quality-checks
-    name: "Integration with smir_pretty"
+    name: "Integration with stable-mir-json"
```

Comment:
This confuses github... the old name was a required check for PRs. Can disable it , then merge and then enable the requirement for the new name.